www/layouts/partials/head.html

```diff
@@ -1,6 +1,5 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width">
-<title>{{ if .IsHome }}{{ site.Title }}{{ else }}{{ printf "%s | %s" .Title site.Title }}{{ end }}</title>
```

Comment:
FTR, this is horrible to read but it produces a nice markup in the resulting HTML document.

I’m kind of an indentation maniac but in this particular case, if your modification is only about coding style, I’d prefer to keep this ugly one-liner and trust `git blame` to remain sure that this is the default Hugo behavior.

Reply:
I need to move it in /layouts/_default/baseof.html so I can override it with a block (the only place where block are allowed in Hugo) otherwise Hugo automatically put "404 Not found" in `.Title` and I haven’t found another way to override it. But I can put all in one line in baseof.html for the markup if it’s ok to keep it in this file ?

Reply:
I pushed the fix, I hope it produces what you expect because I’m not sure what you meant by "trust git blame to remain sure that this is the default Hugo behavior."